lib/gis/timestamp.c

```diff
@@ -425,11 +425,11 @@ int G_has_vector_timestamp(const char *name, const char *layer,
     char ele[GNAME_MAX];
 
     if (layer != NULL)
-        G_snprintf(ele, GNAME_MAX, "%s_%s", GV_TIMESTAMP_ELEMENT, layer);
+        snprintf(ele, GNAME_MAX, "%s_%s", GV_TIMESTAMP_ELEMENT, layer);
```

Comment:
Doesn't this miss include stdio.h?

Reply:
Nice catch, I thought I looked through them all. Updated.